### PR TITLE
Throw error if `controller` is not provided.

### PR DESCRIPTION
**Pull request by JSAssassin**
Addresses #15

### CODE REVIEW

File: lib/ZcapClient.js [hunk 1]
```diff
@@ -140,6 +140,9 @@ export class ZcapClient {
     capability, controller, invocationTarget, expires,
     allowedActions = []
   } = {}) {
+    if(!controller) {
+      throw new TypeError('"controller" must be provided.');
```
Comment (mattcollier) [inline lib/ZcapClient.js:144]:
```suggestion
      throw new TypeError('"controller" is required.');
```
Seems more consistent with the language here: https://github.com/digitalbazaar/ezcap/blob/9fb30bc79fea622c0397a9f69ea4d34b1563a352/lib/ZcapClient.js#L158

**Review by mattcollier — APPROVED** [review]
LGTM, will need approval from @dlongley

File: lib/ZcapClient.js (hunk 1 above)
Comment (dlongley) [inline lib/ZcapClient.js:144]: We may as well get as specific as some of the other checks here:

```suggestion
    if(!(typeof controller === 'string' && controller.includes(':'))) {
      throw new Error(
        '"controller" must be a string expressing an absolute URI.');
    }      
```

**Review by dlongley — APPROVED** [review]

File: lib/ZcapClient.js (hunk 1 above)
Comment (dlongley) [reply lib/ZcapClient.js:144]: Approval conditional on the above being applied.

Comment (JSAssassin) [reply lib/ZcapClient.js]: Updated [83439f1](https://github.com/digitalbazaar/ezcap/pull/16/commits/83439f13b15d0f84ac5f491b19c7b0b2117064f6) and added changelog [2c5bd2c](https://github.com/digitalbazaar/ezcap/pull/16/commits/2c5bd2c7ccb183481356cc286b497f9bcae1b9d5)

**Pull request merged**